spec/features/service_drafts/form_navigation/options_form_nav_spec.rb

```diff
@@ -144,7 +144,7 @@
 
     it 'displays the correct number of required fields' do
       # 1 for each input and output format fields in each paired reformatting
-      expect(page).to have_selector('label.eui-required-o', count: 4)
+      expect(page).to have_selector('label.eui-required-o', count: 9)
```

Comment:
This is passing because the factory draft is full (which means the required icons will be displayed). Ignore this is my assumption about the required icons behavior is incorrect.